internal/commands/create.go

```diff
@@ -70,6 +70,10 @@ func runCreateCommand(path string) error {
 			constraintFileName = fmt.Sprintf("constraint_%s.yaml", violation.Kind())
 		}
 
+		if err := os.MkdirAll(outputDir, os.ModePerm); err != nil {
+			return fmt.Errorf("unable to create output directory: %w", err)
```

Comment:
opinion: I feel including that the operation failed in error messages is slightly redundant when it comes to error wrapping in Go. I view it as more of a stack trace, and you know from the context that whatever you tried to do failed.

For example, if we failed to marshal some JSON that failed after calling 2 other functions: 

`create output dir: get template: marshal json: syntax was invalid`
`unable to create output directory: failed to get template: could not marshal json: syntax was invalid`

Reply:
Fair, and I prefer the brevity of that option as well.